keras_nlp/integration_tests/import_test.py

```diff
@@ -1,20 +0,0 @@
-# Copyright 2021 The KerasNLP Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import keras_nlp
-
-
-# Basic test to make sure the library can be imported.
-def test_import():
```

Comment:
Why remove this? This is the only auto run test that makes sure our __init__.py doesn't error out. And it doesn't seem like you are replacing it here.

Reply:
this is a mistake... I have added them back